cpu/stm32_common/stmclk_l4wb.c

```diff
@@ -72,25 +96,21 @@
 #else
 #error "PLL configuration: PLL R value is invalid"
 #endif
+#endif
 /** @} */
 
 /**
  * @name    Deduct the needed flash wait states from the core clock frequency
  * @{
  */
-#if (CLOCK_CORECLOCK <= 16000000)
-#define FLASH_WAITSTATES            FLASH_ACR_LATENCY_0WS
-#elif (CLOCK_CORECLOCK <= 32000000)
-#define FLASH_WAITSTATES            FLASH_ACR_LATENCY_1WS
-#elif (CLOCK_CORECLOCK <= 48000000)
-#define FLASH_WAITSTATES            FLASH_ACR_LATENCY_2WS
-#elif (CLOCK_CORECLOCK <= 64000000)
-#define FLASH_WAITSTATES            FLASH_ACR_LATENCY_3WS
+#if defined(CPU_FAM_STM32WB)
+#define FLASH_WAITSTATES        ((CLOCK_CORECLOCK - 1) / 18000000U)
 #else
-#define FLASH_WAITSTATES            FLASH_ACR_LATENCY_4WS
+#define FLASH_WAITSTATES        ((CLOCK_CORECLOCK - 1) / 16000000U)
```

Comment:
This looks like an unrelated change. Can you give some pointers why you did that ?

Reply:
Because it depends on the clock speed, the default is right only `CORE_CLOCK` is above some value, the default configuration was with the max of `LASH_ACR_LATENCY_4WS`. But I guess that is unrelated (and actually it should be `CLOCK_AHB`), I'll restore to not introduce the unrelated change here.

Reply:
It was just a cleanup of what was above, it is done for other boards as well.

Reply:
I pushed changing configuration to `CLOCK_AHB` I think current status is cleaner thatn with all the ifdefs.